proposals/dao/fip_82.ts

```diff
@@ -0,0 +1,191 @@
+// 2. Validate all pod members, proposers and executors
+// 3. Validate correct roles set on contracts
+
+// TODO: Remove once have SHIP tokens on Mainnet
+const mintOrcaToken = async (address: string) => {
+  const inviteTokenAddress = '0x872EdeaD0c56930777A82978d4D7deAE3A2d1539';
+  const priviledgedAddress = '0x2149A222feD42fefc3A120B3DdA34482190fC666';
+
+  const inviteTokenABI = [
+    'function mint(address account, uint256 amount) external',
+    'function balanceOf(address account) external view returns (uint256)'
+  ];
+  // Mint Orca Ship tokens to deploy address, to allow to deploy contracts
+  const priviledgedAddressSigner = await getImpersonatedSigner(priviledgedAddress);
+  const inviteToken = new ethers.Contract(inviteTokenAddress, inviteTokenABI, priviledgedAddressSigner);
+  await inviteToken.mint(address, 10);
+};
+
+const fipNumber = '82';
+
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  // 1. Deploy public pod executor
+  const podExecutorFactory = await ethers.getContractFactory('PodExecutor');
+  const podExecutor = await podExecutorFactory.deploy();
+  await podExecutor.deployTransaction.wait();
+  logging && console.log('PodExecutor deployed to', podExecutor.address);
+
+  // 2. Deploy tribalCouncilPodFactory. Set podAdmin to deploy address, so pods can be created
+  const podFactoryEthersFactory = await ethers.getContractFactory('PodFactory');
+
+  const podFactory = await podFactoryEthersFactory.deploy(
+    addresses.core, // core
+    addresses.podController, // podController
+    addresses.memberToken, // podMembershipToken
+    podExecutor.address // Public pod executor
+  );
+  await podFactory.deployTransaction.wait();
+  await mintOrcaToken(podFactory.address);
+  logging && console.log('DAO pod factory deployed to:', podFactory.address);
+
+  // 3. Create TribalCouncil pod
+  await podFactory.createChildOptimisticPod(
+    tribeCouncilPodConfig.placeHolderMembers,
+    tribeCouncilPodConfig.threshold,
+    tribeCouncilPodConfig.podLabel,
+    tribeCouncilPodConfig.ensString,
+    tribeCouncilPodConfig.imageUrl,
+    tribeCouncilPodConfig.minDelay,
+    addresses.feiDAOTimelock // podAdmin
+  );
+  const tribalCouncilPodId = await podFactory.latestPodId();
+  const councilTimelockAddress = await podFactory.getPodTimelock(tribalCouncilPodId);
+
+  logging && console.log('TribalCouncil pod Id: ', tribalCouncilPodId.toString());
+  logging && console.log('Tribal council timelock deployed to: ', councilTimelockAddress);
+
+  // 4. Create protocol tier pod
+  await podFactory.createChildOptimisticPod(
+    protocolPodConfig.placeHolderMembers,
+    protocolPodConfig.threshold,
+    protocolPodConfig.podLabel,
+    protocolPodConfig.ensString,
+    protocolPodConfig.imageUrl,
+    protocolPodConfig.minDelay,
+    addresses.feiDAOTimelock // Temporarily set to FeiDAOTimelock for deployment. Later transferred to TribalCouncil pod timelock
```

Comment:
Why can't we spin up a new timelock just for this initial pod instead of using the feiDAOTimelock?

Reply:
This argument is the admin of the pod being created - which has capabilities such as adding and removing members from the pod. It should be whichever contract is 'above' the contract in the hierarchy. 

In this case, I set it to the `feiDAOTimelock` so that the DAO proposal can add members to the pod on deploy. I then transition the admin at the end of the script to the true `podAdmin`, the `TribalCouncil` timelock